README.md

```diff
@@ -46,11 +46,13 @@ The library is based on node.js which can be natively integrated with external C
 * [Firefox](https://www.mozilla.org/en-US/firefox/products/)
 
 **Server dependencies**: 
-* [YARP](https://github.com/robotology/yarp) (Duh!). With OpenCV!
+* [OpenCV](https://docs.opencv.org/trunk/d7/d9f/tutorial_linux_install.html)
```

Comment:
I'd say that https://github.com/opencv/opencv is a better url.

Reply:
agreed